### PR TITLE
Fix ReflexionHelper class name

### PR DESCRIPTION
**Pull request by mvorisek**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | see title
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | CI must pass

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20707)

### CODE REVIEW

**Comment by mvorisek** [comment]
@PierreRambaud can we merge this small fix incl. the updated `.gitignore`?

File: .gitignore [hunk 1]
```diff
@@ -280,6 +280,7 @@ sftp-config.json
 
 ## Directory-based project format:
 .idea/
+nbproject/
```
Comment (mvorisek) [inline .gitignore]: for Netbeans please :)

Comment (PierreRambaud) [reply .gitignore]: It's not needed, with git you can create a global gitignore file in your home directory
`~/.gitignore` we should only use what we have in this project :thinking:

Comment (mvorisek) [reply .gitignore]: you are right, but as there was `nbproject/private` ignore already, I simplified it and both usecases should be handled

File: .gitignore [hunk 2]
```diff
@@ -302,7 +302,7 @@ crashlytics.properties
 crashlytics-build.properties
 
 ### NetBeans ###
-nbproject/private/
+nbproject/
```
Comment (mvorisek) [inline .gitignore]: we should ignore all projects, not only private files, like for `.idea`

Comment (PierreRambaud) [reply .gitignore]: You must do it in another commit / pull request, it has no aim regarding the title and the description :/

Comment (mvorisek) [reply .gitignore]: PR here - https://github.com/PrestaShop/PrestaShop/pull/21076, this PR also updated to contain only the described change

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Nice catch @mvorisek thank you

**Pull request merged**